pyvcg/smt.py

```diff
@@ -237,6 +237,7 @@ class Logic_Visitor(Visitor):
     def __init__(self):
         self.logics    = set()
         self.functions = set()
+        self._visited_records = set()
```

Comment:
Since none of the other members are private, let's make this one public please for consistency.